### PR TITLE
Fix indentation for skip_row docstring

### PR DESCRIPTION
**Pull request by ulliholtgrave**
Without the newline before the code block intersphinx isn't able to parse the docstring. I also adjusted the indentation of the block and propose to use the python environment for the code block.

**Problem**

[Intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html) isn't able to parse the docstring for `skip_row` which is results in failing documentation builds.
Related issue: #1742 

**Solution**

I added a newline before the code block and optionally adjusted the indentation and code environment of the block.

### CODE REVIEW

**Comment by matthewhegarty** [comment]
Thanks for submitting.  Feel free to add your name to `AUTHORS` and an update to `CHANGELOG.rst` would be appreciated.

File: import_export/resources.py [hunk 1]
```diff
@@ -645,12 +645,13 @@ def skip_row(self, instance, original, row, import_validation_errors=None):
...
-        ::
-          class YourResource(ModelResource):
-            def skip_row(self, instance, original, row, import_validation_errors=None):
-                # Add code here
-                return super().skip_row(instance, original, row,
-                  import_validation_errors=import_validation_errors)
+
+        ..code :: python
+            class YourResource(ModelResource):
+                def skip_row(self, instance, original, row, import_validation_errors=None):
```
Comment (matthewhegarty) [inline import_export/resources.py:651]: pre-commit failing with:

```
import_export/resources.py:651:89: E501 line too long (91 > 88 characters)
```

**Review by matthewhegarty — CHANGES_REQUESTED** [review]
One minor comment re line length

**Comment by ulliholtgrave** [comment]
@matthewhegarty Thanks for the quick response. I think I applied the changes for the check. I also found some more indentation issues that intersphinx wasn't happy about. If you run your setup on `--nitpick` mode this also breaks.

**Comment by ulliholtgrave** [comment]
I already saw the failed test. Will look into it :)

**Comment by matthewhegarty** [comment]
Thanks.  I couldn't run with `nitpicky` - it didn't work for me, even when I installed the dev version of Sphinx.  Please could you raise a new issue which shows how to repeat the break with nitpicky?

**Comment by matthewhegarty** [comment]
Hi - any chance you could look into this so I can get the fix into the next release.
Many thanks

**Comment by matthewhegarty** [comment]
np - thanks for taking this on, much appreciated.

**Comment by matthewhegarty** [comment]
Thank you - will merge

**Pull request merged**